schema_based_on_diagram.sql

```diff
@@ -0,0 +1,52 @@
+-- create patients table 
+CREATE TABLE patients(
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	name varchar(255),
+	date_of_birth DATE
+);
+
+-- create medical_hitories table
+CREATE TABLE medical_histories(
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	admitted_at TIME,
+	patient_id INT ,
+	status varchar(255),
+    FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+-- create invoices table
+CREATE TABLE invoices(
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	total_amount DECIMAL,
+	generated_at TIME,
+	payed_at TIME,
+	medical_history_id INT,
+     FOREIGN KEY (medical_history_id ) REFERENCES medical_histories(id)
+);
+
+
+-- create treatments table
+CREATE TABLE treatments(
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	type varchar(255),
+	name varchar(255)
+);
+
+
+-- add table relationships
+-- create a "join table" called hospitalizations for the medical_histories and treatments tables
+
+CREATE TABLE hospitalizations(
+    ID int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	medical_history_id INT,
+	treatment_id INT,
+	FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+	
+	FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
```

Comment:
- [ ] there are no indexes created for these two foreign keys, please make sure to add them.